lldb/include/lldb/Breakpoint/BreakpointLocation.h

```diff
@@ -69,7 +69,7 @@ class BreakpointLocation
   // The next section deals with various breakpoint options.
 
   /// If \a enabled is \b true, enable the breakpoint, if \b false disable it.
-  void SetEnabled(bool enabled);
+  bool SetEnabled(bool enabled);
```

Comment:
Might be good here to document how this can fail.  If you enable a HW breakpoint after you've used up all the resources with other breakpoints is the only case I can think of, but that's not entirely obvious.  That's the only way this can fail that I can think of.  If there are other ways this could fail, we should return an SBError to distinguish between them.